pennylane/devices/qubit/preprocess.py

```diff
@@ -345,7 +380,6 @@ def _update_config(config: ExecutionConfig) -> ExecutionConfig:
 
 
 def preprocess(
```

Comment:
are we keeping this around? I guess it's used elsewhere eg. adjoint metric tensor

Reply:
Yes I will keep it for this reason.